thirdparty/deto_async/async/internal/qtqueuedinvoker.cpp

```diff
@@ -0,0 +1,21 @@
+#include "qtqueuedinvoker.h"
```

Comment:
Why do we need the "Invoker" class and the "queued" invokers?

Isn't it better to leave just "invoker" and to manage the call type by argument?

Reply:
This is thirdparty :)

Reply:
oah, missed that)